msteams-platform/get-started/glossary.md

```diff
@@ -254,8 +255,8 @@ Common terms and definitions used in Microsoft Teams developer documentation.
 | --- | --- |
 | [Tab](../tabs/what-are-tabs.md) | Tabs are client-aware webpages embedded in Microsoft Teams, Outlook, and Microsoft 365 that point to domains declared in app manifest. You can add it as part of a channel inside a team, group chat, or personal app for an individual user. |
 | [Tab chat](../tabs/how-to/conversational-tabs.md) | A type of tab that lets a user have a focused conversation experience in dynamic tabs. |
-| [Task modules](../task-modules-and-cards/what-are-task-modules.md) | A feature of Teams app to create modal pop-up for completing tasks, displaying videos, or dashboard. <br> **See also**: [Adaptive Card](#a) |
-| [Task info](../task-modules-and-cards/task-modules/invoking-task-modules.md#the-taskinfo-object) | The `TaskInfo` object contains the metadata for a task module.|
+| [Task modules](../task-modules-and-cards/what-are-task-modules.md) (referred as dialogs in TeamsJS v.2.0)| A feature of Teams app to create modal pop-up for completing tasks, displaying videos, or dashboard. <br> **See also**: [Adaptive Card](#a) |
```

Comment:
Same thing for v2--could we instead say "TeamsJS v2.x" to cover all minor versions?

Reply:
Updated